lib/tests/streamlit/watcher/LocalSourcesWatcher_test.py

```diff
@@ -76,8 +76,9 @@ def setUp(self):
             except:
                 pass
 
+    # TODO re-enable
```

Comment:
Todo: Re-enable

Reply:
#1221  was merged, so you can re-enable this